### PR TITLE
Add multi-vector indexing sample app

### PR DESCRIPTION
**Pull request by jobergum**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: multi-vector-indexing/README.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+Verify that configuration service (deploy api) is ready
+
+<pre data-test="exec">
+$ vespa status deploy --wait 300
+</pre>
+
+Download this sample application 
+
+<pre data-test="exec">
+$ vespa clone wikipedia-search my-app && cd my-app
```
Comment (kkraune) [inline multi-vector-indexing/README.md:77]: is it wikipedia-search or multi-vector-indexing?

File: multi-vector-indexing/README.md [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+ }
+```
+This index corresponds to the following paragraph:
+```
+"In railway timetables 24:00 means the \"end\" of the day. For example, a train due to arrive at a station during the last minute of a day arrives at 24:00; but trains which depart during the first minute of the day go at 00:00."
+```
+The [tensor presentation format](search/query-profiles/default.xml) is overridden in
+this sample application to shorten down the output. 
+
+### Hybrid search and ranking
```
Comment (kkraune) [inline multi-vector-indexing/README.md:159]: This section is great, but I would like a bit more on the multi-vector aspect of it. I assume we will add links here to doc/guides for more information. so this is good for now. Later there should be a heading with "multiple vectors" or the likes

**Review by kkraune — APPROVED** [review]
💯 ground-breaking!

File: multi-vector-indexing/README.md (hunk 2 above)
Comment (jobergum) [reply multi-vector-indexing/README.md:159]: Trying to keep this short and sweet.

File: multi-vector-indexing/schemas/wiki.sd [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    function avg_paragraph_similarity() {
+      expression {
+        reduce(
+          sum(l2_normalize(query(q),x) * l2_normalize(attribute(paragraph_embeddings),x),x),
+          avg,
+          p
+        )
+      }
+    }
+    function avg_paragraph_similarity() {
```
Comment (geirst) [inline multi-vector-indexing/schemas/wiki.sd:67]: This function is defined two times. See L58.

**Review by geirst — COMMENTED** [review]
👍

File: multi-vector-indexing/schemas/wiki.sd (hunk 3 above)
Comment (jobergum) [reply multi-vector-indexing/schemas/wiki.sd]: Thanks

**Pull request merged**